pySDC/projects/PinTSimE/battery_model.py

```diff
@@ -76,6 +76,7 @@ def main():
     description['sweeper_params'] = sweeper_params  # pass sweeper parameters
     description['level_params'] = level_params  # pass level parameters
     description['step_params'] = step_params
+    description['convergence_controllers'] = {SwitchEstimator: {}}
```

Comment:
I started with a list, but this was not working anymore: https://github.com/Parallel-in-Time/pySDC/blob/68c8728fd838e602ff5d2d2373c5150a7098e55e/pySDC/core/Step.py#L128 
Also, I like this, because it allows to pass specific parameters to specific convergence controllers (in this case none).
How else would you do this? I could also pass the same dict to all convergence controllers if you really prefer. But lists in your convention are for multi level, and this dict is not about multi level.